### ISSUE TITLE
upload/download speed uses 2 rows

### ISSUE THREAD

**Issue by tessus** [transmission/transmission#6375]
### What is the issue?

<img width="247" alt="image" src="https://github.com/transmission/transmission/assets/223439/1523b021-411c-41bc-ad85-d86cd61969fa">

Once again there's something off with the UI elements. The up and download speed indicators wrap and are displayed in 2 rows. 

The previous fix doesn't work anymore:

```
sed -i 's/speed-up-label{text-align:right;width:70px}/speed-up-label{text-align:right;width:100px;white-space:nowrap}/g'
```

Update: Sorry, the fix still works. It is now in a different file, which I missed.

### Which application of Transmission?

Web client

### Which version of Transmission?

4.0.5

**Comment by killemov** [comment]
The horizontal space does not appear to be sufficient for the text. In this case wrapping is ugly. I would implement a combination of align-left, nowrap and clip (not ellipsis) so the text for download speed is cut off just before the upload arrow. Hotfixing the style with sed should not be necessary unless the devs think the current way of displaying is preferable.

**Comment by tessus** [comment]
`sed` is just a way to hotfix the current version. In reality this should have already made it into the source. And as far as I can remember it was fixed at one point, unless I am mistaken. 
Either way, changing the width to 100px and using nowrap is the fix. There's space more than enough so rstricting it to 70px is just wrong.

**Comment by dareiff** [comment]
So it is _not_ fixed in this? (For 4.1) https://github.com/transmission/transmission/pull/5828

I’ll do some investigating!

**Comment by tessus** [comment]
@dareiff I haven't tested this PR, but according to the title is has to do with narrow viewports. The issue I described even happens when I have the browser in fullscreen on a Retina display with a resolution of 2880x1800.

**Comment by tearfur** [comment]
> So it is _not_ fixed in this? (For 4.1) #5828
> 
> I’ll do some investigating!

It is, the commit was not backported to `4.0.x` though.

**Comment by dareiff** [comment]
> > So it is _not_ fixed in this? (For 4.1) #5828
> > I’ll do some investigating!
> 
> It is, the commit was not backported to `4.0.x` though.

(Can confirm)
